### PR TITLE
Strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

**Review by enricocid — APPROVED** [review]

**Comment by enricocid** [comment]
Thank You. Great improvements. 👍🏻 :)

**Comment by RickyM7** [comment]
@enricocid The "Turn On" and "Turn Off" options have to be reversed or else things will get confusing.

**Comment by enricocid** [comment]
> @enricocid The "Turn On" and "Turn Off" options have to be inverted or else things will get confusing.

You are right. 
error_no_browser too was wrong as this string is referred to web browsers

Anyway I've fixed :) Thanks

**Comment by RickyM7** [comment]
I think in the string `<string name="loved_song_remove">%1s (%2s) will be deleted from loved songs. Continue?</string>` "deleted" can be changed by "removed" as well.

**Comment by comradekingu** [comment]
@RickyM7 Are you sure I don't have it right with the off and on?
Now it is in sequence, and according to what the key says for each string.

**Comment by RickyM7** [comment]
@comradekingu The point is that it is in the legend of the option, saying what will happen if we click on it, for example, if the option "Audio Focus" is enabled, the legend will be, Turn Off Audio Focus Management", which is what will happen if you click the option.
Note: sorry for my bad english, I hope you can understand what I wanted to say.

**Comment by enricocid** [comment]
> @RickyM7 Are you sure I don't have it right with the off and on?
> Now it is in sequence, and according to what the key says for each string.

https://material.io/design/platform-guidance/android-settings.html#label-and-secondary-text

You can:
**1. Explain what happens when a setting is enabled or disabled.**
2. or describe the current status

The first is the simpler one (description of the status are generally too wordy)

See the Bluetooth example :)
